dev/scaletesting/codehostcopy/github.go

```diff
@@ -37,21 +38,41 @@ func NewGithubCodeHost(ctx context.Context, def *CodeHostDefinition) (*GithubCod
 }
 
 func (g *GithubCodeHost) ListRepos(ctx context.Context) ([]*store.Repo, error) {
-	opts := github.RepositoryListByOrgOptions{
-		ListOptions: github.ListOptions{},
-	}
 	var repos []*github.Repository
-	for {
-		rs, resp, err := g.c.Repositories.ListByOrg(ctx, g.def.Path, &opts)
-		if err != nil {
-			return nil, err
+
+	if strings.HasPrefix(g.def.Path, "@") {
+		// If we're given a user and not an organization, query the user repos.
+		opts := &github.RepositoryListOptions{
+			ListOptions: github.ListOptions{},
+		}
+		for {
+			rs, resp, err := g.c.Repositories.List(ctx, strings.Replace(g.def.Path, "@", "", 1), opts)
```

Comment:
```suggestion
			rs, resp, err := g.c.Repositories.List(ctx, strings.Replace(g.def.Path, "@", "", 1), &opts)
```
If the previous change is accepted 👯

Reply:
Yes of course!